package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "NODE_ENV=production lerna run build",
+    "create-releases": "lerna version --no-push --conventional-commits --message 'chore(release): %s'",
```

Comment:
@dierat, looks like I accidentally buried your question here:

> Does this only work when bumping the patch version (not minor/major)?

No, this on works on any version bump type, as determined by the conventional commit messages (fix, feat, etc)

Reply:
Cool, thanks!